DTML.EduBot/Constants/BotEntities.cs

```diff
@@ -14,5 +14,44 @@ public static class BotEntities
         public static string Color = "color";
         public static string Subject = "subject";
         public static string encyclopediaOrganization = "builtin.encyclopedia.organization.organization";
+        public static string TranslateTargetLanguage = "Translate.TargetLanguage";
+        public static string TranslateText = "Translate.Text";
+        public static string Animal = "animal";
+        public static string Drink = "drink";
+        public static string Action = "action";
+        public static string EndConversation = "endconversation";
+        public static string StartConversation = "startconversation";
+        public static string Family = "family";
+        public static string Food = "food";
+        public static string Gibberish = "gibberish";
+        public static string Language = "language";
+        public static string Next = "next";
+        public static string Place = "place";
+        public static string Previous = "previous";
+        public static string StartOver = "startover";
+        public static string AboutMe = "aboutme";
+        public static string Hobby = "hobby";
+        
+
+        //TODO: what is datetimeV2's entity value?
+        public static string DateTimeV2 = "";
+        //TODO: what are emotion's entity values?
+        public static string Emotion = "emotion";
+        public static string EmotionNegative = "emotion::negative";
+        public static string EmotionPositive = "emotion::positive";
+        public static string EmotionNeutral = "emotion::neutral";
+        //TODO: what is money's entity value?
+        public static string Money = "";
+        //TODO: what is number's entity value?
```

Comment:
Can you resolve all todo comments? This looks like something that should be completed for this to be called feature complete.